make/addons.mk

```diff
@@ -3,12 +3,12 @@
 
 export CALICO_VERSION := v3.26.4
 export CILIUM_VERSION := 1.15.0
-export NODE_FEATURE_DISCOVERY_VERSION := 0.15.2
+export NODE_FEATURE_DISCOVERY_VERSION := v0.15.2
```

Comment:
These two values changed because mindthegap wasn't able to get the tags as they were previously, but with these changes it was